src/components/TaskListItem.js

```diff
@@ -213,7 +214,13 @@ class TaskListItem extends Component {
               <Text numberOfLines={ 1 } style={ textStyle }>{ task.address.streetAddress }</Text>
               <HStack alignItems="center">
                 <Text pr="2" style={ textStyle }>{ moment(task.doneAfter).format('LT') } - { moment(task.doneBefore).format('LT') }</Text>
-                { task.comments && task.comments.length ? <Icon as={FontAwesome} name="comments" size="xs"/> : null }
+                { task.comments && task.comments.length ? <Icon mr="2" as={FontAwesome} name="comments" size="xs"/> : null }
+                {
+                  task.metadata && task.metadata.payment_method &&
```

Comment:
`task.metadata.payment_method` may be an empty string, so it would be better to use a ternary operator to avoid [this bug ](https://github.com/facebook/react-native/issues/29980)

Reply:
Ok actually the bug doesn't happen even with an empty string, no problem 🙂